client/src/components/web-extension/RecoveryPhrase/index.jsx

```diff
@@ -26,38 +28,64 @@ const RecoveryPhrase = () => {
 	const words = keyphrase ? keyphrase.split(' ') : [];
 
 	return (
-		<div className="cd_we_recovery-keyphrase">
-			<div className="cd_we_recovery-keyphrase__wrapper">
-				<ul className="cd_we_recovery-keyphrase__column">
-					{words?.map((word, index) => (
-						<li className="cd_we_recovery-keyphrase__word" key={`left-${index}`}>
-							<span className="counter">{index + 1}</span>
-							<span className="value">{word}</span>
-						</li>
-					))}
-				</ul>
-				<div className="cd_we_recovery-keyphrase__footer">
-					<DownloadButton
-						className="cd_we_recovery-keyphrase__copy-btn"
-						text={keyphrase}
-						fileName="key_phrase.txt"
-					/>
+			<div className="cd_we_recovery-keyphrase">
+				<div className="cd_we_recovery-keyphrase__wrapper">
+					<div className={clsx({
+						'cd_we_recovery-keyphrase__blur': isBlurred,
+					})}
+			onClick={() => setIsBlurred(false)}
+					>
+						<ul className={
+							clsx('cd_we_recovery-keyphrase__column', {
+								'cd_we_recovery-keyphrase__column--blurred': isBlurred,
+							})
+						}
+						>
+							{words?.map((word, index) => (
+								<li className="cd_we_recovery-keyphrase__word" key={`left-${index}`}>
+									<span className="counter">{index + 1}</span>
+									<span className="value">{word}</span>
+								</li>
+							))}
+						</ul>
+						{
+							isBlurred && (
+								<div className="cd_we_recovery-keyphrase__blur-overlay">
+									Click to reveal secret recovery phrase
```

Comment:
Should be "Reveal secret recovery phrase"

Reply:
Maybe `Click` is more clear for user :D